zebra-chain/src/orchard/keys/tests.rs

```diff
@@ -39,21 +80,20 @@ proptest! {
 
         let spend_authorizing_key = SpendAuthorizingKey::from(spending_key);
         // Test ConstantTimeEq, Eq, PartialEq
-        assert!(spend_authorizing_key == <[u8; 32]>::from(spend_authorizing_key));
+        assert!(spend_authorizing_key == spend_authorizing_key.clone());
```

Comment:
Nit: should these checks use `assert_eq!`?

```suggestion
        assert_eq!(spend_authorizing_key, spend_authorizing_key.clone());
```

Reply:
I think it should be equivalent? I've changed it in e59e540

Reply:
The `assert_eq!` message shows both values, so we should prefer `assert_eq!` and `assert_ne!` where possible.